dookug-common/dookug-common-pdfsignature/src/main/java/hu/icellmobilsoft/dookug/engine/pdfbox/signing/SignatureGenerator.java

```diff
@@ -261,8 +260,8 @@ public byte[] sign(InputStream content) throws IOException {
             CMSPrivateKey cmsPrivateKey = privateKeyLoader.privateKeyResolver(signatureProfile);
             CMSSignedDataGenerator gen = new CMSSignedDataGenerator();
             X509Certificate cert = (X509Certificate) cmsPrivateKey.getCertificateChain()[0];
-            ContentSigner shaSigner = new JcaContentSignerBuilder(SHA256_WITH_RSA).build(cmsPrivateKey.getPrivateKey());
-
+            ContentSigner shaSigner = new JcaContentSignerBuilder(signatureProfile.getPdfBoxSignatureAlgorithm())
```

Comment:
mi lesz ha olyan signaturet ad be ami nem letezik? az hogy fog latszodni vajon, mert ugy nezem exceptiont fog dobni: https://github.com/bcgit/bc-java/blob/main/pkix/src/main/java/org/bouncycastle/operator/DefaultSignatureAlgorithmIdentifierFinder.java#L745
nincs most valami ellenorzo rész amikor felfut a service hogy ez a config egyebkent valos algoritmust akar hasznalni?